.github/workflows/tox-test.yml

```diff
@@ -3,7 +3,7 @@ name: Tox tests
 on: [push, pull_request]
 
 jobs:
-  py27:
+  py38:
```

Comment:
I don't think you need to add this, because the 'coverage' job is already testing python3.  The coverage job does all the same steps but also uploads coverage info at the end.  We don't need both of them, you should be able to just drop the py27 job and not add any replacement.